### PR TITLE
EPMDJ-5900 Added maven sample of application using drill

### PR DESCRIPTION
**Pull request by Mazel-Tovr**

### CODE REVIEW

File: samples/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+2. [Versions](#Versions)
+3. [Common Start and register app](#Common-Start-and-register-app)
+4. [Using Test2Code plugin](#Using-Test2Code-plugin)
+    - [Start Autotests](#Start-Autotests)
+  <!--  - [Manual Browser extension](#Manual-Browser-extension)--->
+
+## Preparation env for latest versions
+
+- <a href="https://www.docker.com/products/docker-desktop">Download docker</a>
+- Download latest version of drill agent for your OS
```
Comment (LikhanovMaxim) [inline samples/README.md]: here you had links on artifactory, why did you remove them?

File: samples/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- <a href="https://www.docker.com/products/docker-desktop">Download docker</a>
+- Download latest version of drill agent for your OS 
+- Unzip it
+- Go to folder /maven-sample/spring-petclinic in pom.xml file set value to "pathToAgent" property
+
+## Versions
+
+|Component   | Release | Stable | Latest | Description|
+|:----------:|:-----:|:-------:|:---------:|:----|
+|Admin-ui| <a href="https://github.com/Drill4J/admin-ui/releases/tag/v0.6.0">v0.6.0</a> | - | <a href="https://github.com/Drill4J/admin-ui/releases/tag/v0.7.0-155">v0.7.0-155</a> | Drill-Ui <a href="https://github.com/Drill4J/admin-ui">learn more</a>|
```
Comment (LikhanovMaxim) [inline samples/README.md]: We have versions in https://drill4j.github.io/stable-versions/ 
We don't need to duplicate this information, it should be update info in one place

File: samples/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+|Java agent| <a href="https://github.com/Drill4J/java-agent/releases/tag/v0.6.0">v0.6.0</a> | - | <a href="https://github.com/Drill4J/java-agent/releases/tag/v0.7.0-24">v0.7.0-24</a> | Java agent <a href="https://github.com/Drill4J/java-agent">learn more</a> |
+|Autotest agent| <a href="https://github.com/Drill4J/autotest-agent/releases/tag/v0.6.0">v0.6.0</a> | - | <a href="https://github.com/Drill4J/autotest-agent/releases/tag/v0.12.0">v0.12.0</a> | Autotest agent <a href="https://github.com/Drill4J/autotest-agent">learn more</a> |
+|Browser extantion| <a href="https://github.com/Drill4J/browser-extension/releases/tag/v0.3.16">v0.3.16</a> | - | <a href="https://github.com/Drill4J/browser-extension/releases/tag/v0.3.27">v0.3.27</a> | Browser extantion <a href="https://github.com/Drill4J/browser-extension">learn more</a> |
+
+## Common Start and register app
+- Launch docker
+- Go to folder /admin run console and write ```docker-compose up```
+- Launch "spring-petclinic" application applictation took from <a href="https://github.com/Drill4J/spring-petclinic">here</a>
```
Comment (LikhanovMaxim) [inline samples/README.md]:
```suggestion
- Launch <a href="https://github.com/Drill4J/spring-petclinic">spring-petclinic</a>
```

File: samples/README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    - Set package ![alt text](img/registerPackage.png)
+    - Add test2code plugin ![alt text](img/registerTest2code.png)
+   
+#### Links
+
+- Admin UI: http://localhost:3000
+- Admin Backend Swagger: http://localhost:8090
+
+## Using Test2Code plugin
+### Start Autotests
```
Comment (LikhanovMaxim) [inline samples/README.md:50]: add the list of autotest engines that you use

File: samples/admin/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+version: "3"
+
+services:
+  admin-ui:
+    # it's a lates version of drill ui
+    image: drill4j/admin-ui
+    # To find the version you need, follow the link  https://hub.docker.com/r/drill4j/admin-ui/tags
```
Comment (LikhanovMaxim) [inline samples/admin/docker-compose.yml]:
```suggestion
    # it's the latest version of drill ui
     # The list of other versions here https://hub.docker.com/r/drill4j/admin-ui/tags
    image: drill4j/admin-ui
   
```

File: samples/admin/docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    ports:
+      - 3000:8080
+    environment:
+      - REACT_APP_API_HOST=drill-admin:8090
+  admin:
+    # it's a lates version of drill backend
```
Comment (LikhanovMaxim) [inline samples/admin/docker-compose.yml:13]: the same

File: samples/maven-sample/spring-petclinic/pom.xml [hunk 7]
```diff
@@ -0,0 +1,248 @@
...
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <!-- Web dependencies -->
+        <webjars-bootstrap.version>3.3.6</webjars-bootstrap.version>
+        <webjars-jquery-ui.version>1.11.4</webjars-jquery-ui.version>
+        <webjars-jquery.version>2.2.4</webjars-jquery.version>
+        <wro4j.version>1.8.0</wro4j.version>
+        <!-- For example D:\Github\Drill-Samples\maven-sample\spring-petclinic\agent        -->
+        <pathToAgent>"Path to agent"</pathToAgent>
```
Comment (LikhanovMaxim) [inline samples/maven-sample/spring-petclinic/pom.xml]: are those quotes need? Why don't use default value with TODO change? For example:
```suggestion
        <pathToAgent>"D:\Github\Drill-Samples\maven-sample\spring-petclinic\agent"</pathToAgent>
```

**Review by LikhanovMaxim — CHANGES_REQUESTED** [review]

File: samples/README.md (hunk 2 above)
Comment (Mazel-Tovr) [reply samples/README.md]: do you suggest removing this table completely or removing only the stable versions ?

File: samples/admin/docker-compose.yml (hunk 6 above)
Comment (LikhanovMaxim) [reply samples/admin/docker-compose.yml:13]: fix as the above
image

File: samples/README.md (hunk 4 above)
Comment (LikhanovMaxim) [reply samples/README.md:50]: add selenium

File: samples/README.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+3. [Common Start and register app](#Common-Start-and-register-app)
+4. [Using Test2Code plugin](#Using-Test2Code-plugin)
+    - [Start Autotests](#Start-Autotests)
+  <!--  - [Manual Browser extension](#Manual-Browser-extension)--->
+
+## Preparation env for latest versions
+
+- <a href="https://www.docker.com/products/docker-desktop">Download docker</a>
+- Download the latest version of drill agent for your OS 
+- Download actual version of drill agent for your OS
```
Comment (LikhanovMaxim) [inline samples/README.md]: remove this line
```suggestion
- Download the latest version of drill agent for your OS
```

File: samples/README.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
+# Drill-Samples
+
+## Table of contents
+1. [Preparation env for latest versions](#Preparation-env-for-latest-versions)
+2. [Versions](#Versions)
```
Comment (LikhanovMaxim) [inline samples/README.md]: remove or add todo

**Review by LikhanovMaxim — CHANGES_REQUESTED** [review]
